compiler/AST/TryStmt.cpp

```diff
@@ -52,30 +70,61 @@ void TryStmt::accept(AstVisitor* visitor) {
     if (_body) {
       _body->accept(visitor);
     }
+
+    for_alist(c, _catches) {
+      CatchStmt* catchStmt = toCatchStmt(c);
+      catchStmt->accept(visitor);
+    }
+
     visitor->exitTryStmt(this);
   }
 }
 
-Expr* TryStmt::copy(SymbolMap* map, bool internal) {
-  return NULL;
-}
-
-Expr* TryStmt::copyInner(SymbolMap* map) {
-  return NULL;
+TryStmt* TryStmt::copyInner(SymbolMap* map) {
+  TryStmt* copy = new TryStmt(_tryBang, COPY_INT(_body), NULL);
+  for_alist(c, _catches) {
+    copy->_catches.insertAtTail(c->copy());
```

Comment:
I believe this should use `COPY_INT(c)` rather than `c->copy()`. Could you try it & accept it if your tests pass?